README.md

```diff
@@ -1,3 +1,22 @@
 # Rulma
 
 React components for [Bulma](https://github.com/jgthms/bulma)
+
+## Getting started
+#### Install
+```bash
+npm install --save rulma
+```
+
+#### Include Bulma CSS
```

Comment:
We already have Bulma as a dependency, so it is actually available for the user. But maybe we should move it to a dev dependency instead?